spec/system/vendor_system_spec.rb

```diff
@@ -13,11 +13,50 @@
       @third = create(:vendor, business_name: "Cde")
       visit vendors_path
     end
+
     it "should have the vendor names in alphabetical order" do
       expect(page).to have_xpath("//table//tr", count: 4)
       expect(page.find(:xpath, "//table/tbody/tr[1]/td[1]")).to have_content(@first.business_name)
       expect(page.find(:xpath, "//table/tbody/tr[3]/td[1]")).to have_content(@third.business_name)
     end
+
+    it "should have a deactivate button for each active vendor" do
```

Comment:
Same here. In general anything that doesn't have interaction should be a request spec.